src/box/lua/config/annotations.lua

```diff
@@ -0,0 +1,2162 @@
+local function toline(s)
+    return s:gsub('\n', ''):gsub(' +', ' '):strip()
+end
```

Comment:
Should the description really be a single line? Some of them are quite big: contain some lists and seems like a several paragraph text. How it looks better it an IDE suggestion?

Reply:
The description can now contain multiple paragraphs and also supports arrays (starting with `-`).